charmhelpers/osplatform.py

```diff
@@ -47,3 +47,10 @@ def _get_platform_from_fs():
     for k, v in content.items():
         content[k] = v.strip('"')
     return content["NAME"]
+
+
+# If the unit-test mode is set, the platform is always "ubuntu"
+if not os.environ.get('CHARMHELPERS_IN_UNITTEST', False):
```

Comment:
the docs suggest that setting this env variable to "false" (considering that 'true' is valid to opt in), the doc should say something like "anything that python evaluates to true"